### PR TITLE
Static Pages

### PR DESCRIPTION
**Pull request by AilinSoon**
ready

### CODE REVIEW

**Comment by AilinSoon** [comment]
ready

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 class ApplicationController < ActionController::Base
+ before_action :set_locale
```
Comment (ngocvt-0484) [inline app/controllers/application_controller.rb]: wrong indent
tab 2 space please

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   def set_locale
-      I18n.locale = params[:locale] || I18n.default_locale
-      #cureent_user.locale
-      #request.env["HTTP_ACCEPT_LANGUAGE"]
+     I18n.locale = params[:locale] || I18n.default_locale
   end
 
   def default_url_options(options = {})
-    { locale: I18n.locale }.merge options
+     { locale: I18n.locale }.merge options
```
Comment (ngocvt-0484) [inline app/controllers/application_controller.rb]: same
wrong indent

File: config/locales/en.yml [hunk 3]
```diff
@@ -1,33 +1,10 @@
...
-  hello: "Hello world"
+  home: "Home"
+  heading: "Course Registration System"
+  help: "Help"
+  t_p1: "This is the home page for the course registration application."
+  t_p2: " This is the help page for the course registration application."
+  log_in: "Log in"
+  c_r: "Course Registration"
+  ja: "日本語"
+
```
Comment (ngocvt-0484) [inline config/locales/en.yml]: remove blank line

**Comment by AilinSoon** [comment]
ready

**Pull request merged**